app/src/main/java/org/coepi/android/ui/alertsdetails/AlertsDetailsViewModel.kt

```diff
@@ -27,19 +31,38 @@ class AlertsDetailsViewModel(
     //TODO Show when the symptoms were reported. Maybe the earliestSymptom Date?
     val reportedTime: LiveData<String> = just(" ").toLiveData()
 
-    private fun Alert.toViewData(): List<AlertDetailsSymptomViewData> =
-        emptyList()
-//        report.ids.map { it.toViewData() }
+    val symptomList = symptomList(args.report.report)
+
+    @SuppressLint("DefaultLocale")
+    private fun symptomList(report: PublicReport): String {
+        val cough = when (report.coughSeverity) {
```

Comment:
We made extensions with @laurenipsum365 for these strings. To add the bullets, map over those strings.

This is another reason to merge early, so it's easier to sync. Either copy or wait until her PR is merged. Put them in a central place, like ui > extensions folder. The file can be called `PublicReportExtensions`.